avocado/core/task/statemachine.py

```diff
@@ -79,7 +79,7 @@ async def triage(self):
         except IndexError:
             return
 
-        requirements_ok = await self._spawner.spawn_task(runtime_task)
+        requirements_ok = await self._spawner.check_task_requirements(runtime_task)
         if requirements_ok:
```

Comment:
Hi @clebergnu, good catch! I was just about to ask you about this and why we use the same status from `spawn_task` in both cases when we check for requirements and when we check if the task started successfully.

Reply:
And this was causing tests to run twice... which made me worry about the new code performance... finding the culprit saved my mood! :)